### PR TITLE
Legacy style cleanup

### PR DESCRIPTION
**Pull request by phalkunz**
Issue: https://github.com/silverstripe/silverstripe-admin/issues/201

### CODE REVIEW

**Comment by chillu** [comment]
@lukereative @phalkunz Have you used the frameworktest module to test some form field variations that are not in the default installation?

File: client/src/styles/legacy/_mixins.scss [hunk 1]
```diff
@@ -119,19 +42,21 @@ Used in side panels and action tabs
...
-		/*
+		/**
 		 * Fields are more compressed in some areas compared to the
 		 * main content editing window so the below alters the internal
 		 * spacing of the fields so we can move that spacing to between
 		 * the form fields rather than padding
 		 */
 		border-bottom:none;
-		box-shadow: none;
+    box-shadow: none;
```
Comment (chillu) [inline client/src/styles/legacy/_mixins.scss]: Wrong indentation? Go through the pull request again please and ensure we're following our coding conventions there (at least in terms of indentation).

Comment (phalkunz) [reply client/src/styles/legacy/_mixins.scss]: @chillu Editor config says `indent_style = space`. The indentation should use spaces instead but I don't want to replace tabs with spaces everywhere.

Comment (phalkunz) [reply client/src/styles/legacy/_mixins.scss]: @chillu I've places tabs with 2 spaces for indentation now

**Comment by chillu** [comment]
We decided to wait with this merge until 4.1 is branched off (since we don't want a change with relatively high chance of regressions to go straight into an 4.1.0-rc1 that's happening this week)

**Pull request merged**